README.md

```diff
@@ -263,16 +263,16 @@ docker run --gpus all -it --rm  -v $(pwd):/home/app/mava -w /home/app/mava insta
     * Install core dependencies:
 
     ```bash
-    pip install id-mava[tf,reverb,launchpad]
+    pip install id-mava[tf,reverb]
     ```
 
     * Or for the latest version of mava from source (**you can do this for all pip install commands below for the latest depedencies**):
 
     ```bash
-    pip install git+https://github.com/instadeepai/Mava#egg=id-mava[reverb,tf,launchpad]
```

Comment:
Just a minor quesiton, does launchpad get installed somewhere else now? Or why is it removed?

Reply:
I thought that it would have been installed by one of the other dependencies i.e. tf or reverb but does not seem to be the case

Reply:
we are getting launchpad via the acme install now

Reply:
Thank you @AsadJeewa :)